src/main/java/com/github/dockerjava/api/model/Statistics.java

```diff
@@ -19,6 +21,17 @@
     @JsonProperty("read")
     private String read;
 
+    /**
+     * @since Docker Remote API 1.21
+     */
+    @CheckForNull
+    @JsonProperty("networks")
+    private Map<String, Object> networksStats;
```

Comment:
Is there any chance to include this change in release 2.1.3? I need this feature working, and I suppose that 3.0.0 is still not released soon.